guides/doc-Monitoring_Project/topics/con_metrics-data-retention.adoc

```diff
@@ -0,0 +1,13 @@
+[id='metrics-data-retention_{context}']
+= Metrics Data Retention
+
+The storage capacity required by PCP data logging is determined by the following factors:
+
+* the metrics being logged,
```

Comment:
```suggestion
* the metrics being logged.
```
And below.

Reply:
you are saying bullet points should always end with a period? should they then be also capitalized, even if they are not full sentences?

Reply:
I would suggest the later one. Capitalize with Period. You may need some rewording in points. For your [reference](https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/con_managing-locations.adoc?plain=1)